meta-lmp-base/classes/lmp-staging.bbclass

```diff
@@ -5,15 +5,15 @@
 #
 # Copyright 2022 (C) Foundries.IO LTD
```

Comment:
We need to add 2023 here as well.

Reply:
I think is better to remove the year instead of have a requirement to update it every year.

Reply:
Copyright is related to the year range that is valid for it, so we can have it 2022-2023 atm.

Reply:
updated the year, in 2024 will remove this part.